SDK/bundles/org.eclipse.embedcdt.debug.gdbjtag.core/src/org/eclipse/embedcdt/debug/gdbjtag/core/dsf/GnuMcuFinalLaunchSequence.java

```diff
@@ -389,6 +393,39 @@ public void stepGnuMcuStart(RequestMonitor rm) {
 		queueCommands(commandsList, rm);
 	}
 
+	@Execute
+	public void stepRegisterService(final RequestMonitor rm) {
```

Comment:
This step not dealing with register service creation/registration so rename it.

Reply:
Changes are done